src/plugin/plugin-manager.js

```diff
@@ -82,15 +82,29 @@ export default class PluginManager {
       throw new Error(Error.Severity.RECOVERABLE, Error.Category.PLAYER, Error.Code.RUNTIME_ERROR_NOT_REGISTERED_PLUGIN, name);
     }
     let pluginClass = PluginManager._registry.get(name);
-    if (pluginClass && pluginClass.isValid()) {
+    this._maybeDisablePlugin(pluginClass, config.disable);
+    if (pluginClass && pluginClass.isValid() && !pluginClass.disabled) {
```

Comment:
why not just
`if (pluginClass && pluginClass.isValid() && !config.disable) { ....` ?